README.md

```diff
@@ -26,10 +31,9 @@ $ conda install cookiecutter
 ### To start a new project, run:
 ------------
 
-    cookiecutter https://github.com/drivendata/cookiecutter-data-science
+    cookiecutter https://github.com/ukgovdatascience/cookiecutter-data-science-gds
 
 
-[![asciicast](https://asciinema.org/a/244658.svg)](https://asciinema.org/a/244658)
 
 
 ### The resulting directory structure
```

Comment:
Not sure if necessary, but this structure is missing descriptions for:

- `.gitignore`
- `test_environment.py`

Reply:
Straightforward changes done.

As for `create_environment` rule, I do not think `conda`  is the default environment manager at GDS but it is worth keeping it in as an option. I would also add `pipenv` (it is the one I use) as a third option (the second one being `pip` + `virtualenv` + `virtualenvwrapper`). 

I would then remove the option for python 2 here as well (see changes below following your comment).

Reply:
Let's have a separate discussion outside of this PR about `conda`/`pip`/`pipenv` etc. Marking this as **Resolved** here, but have added a card on Trello.